xcodeproj/internal/xcodeproj_tests.bzl

```diff
@@ -1,7 +1,47 @@
 """Tests for the `xcodeproj` rule."""
 
-load("//test/fixtures:fixtures.bzl", "fixtures_transition")
+load("@bazel_skylib//lib:paths.bzl", "paths")
 load("//xcodeproj:xcodeproj.bzl", "XcodeProjOutputInfo")
+load(":fixtures.bzl", "fixtures_transition")
+
+# MARK: - xcodeproj_tests
+
+def _from_fixture(
+        target_under_test,
+        basename = None,
+        expected_spec = None,
+        expected_xcodeproj = None):
+    if target_under_test == None:
+        fail("Need to specify the target under test.")
+    if target_under_test.find(":") < 0:
+        target_under_test += ":xcodeproj"
+
+    test_target_parts = target_under_test.split(":")
+    pkg = test_target_parts[0]
```

Comment:
I don't think we want to default this.

Can we turn it into a label if it's not, then use it's APIs? If not, we can fail if it's not a fully qualified string.